Android/Samples/libraries/kml/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.library'
+apply plugin: "com.github.sherter.google-java-format"
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'consumer-rules.pro'
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Prefer ordering implementation, then testImplementation and last androidTestImplementation (if you need it; I don't believe you do here)

Reply:
And you can remove the jar import.